ui/packages/consul-ui/app/templates/dc/acls/tokens/edit.hbs

```diff
@@ -96,18 +96,10 @@ as |dc partition nspace item create|}}
       </BlockSlot>
       <BlockSlot @name="content">
           {{#if (token/is-legacy item)}}
-            <Notice
-              @type="info"
-            as |notice|>
-              <notice.Header>
-                <h2>Update</h2>
-              </notice.Header>
-              <notice.Body>
-                <p>
-                  We have upgraded our ACL system by allowing you to create reusable policies which you can then apply to tokens. Don't worry, even though this token was written in the old style, it is still valid. However, we do recommend upgrading your old tokens to the new style. Learn how in our <a href="{{env 'CONSUL_DOCS_URL'}}/guides/acl-migrate-tokens.html" target="_blank" rel="noopener noreferrer">documentation</a>.
-                </p>
-              </notice.Body>
-            </Notice>
+            <Hds::Alert @type='inline' class='mb-6' as |A|>
+              <A.Title>Update</A.Title>
+              <A.Description>We have upgraded our ACL system by allowing you to create reusable policies which you can then apply to tokens. Don't worry, even though this token was written in the old style, it is still valid. However, we do recommend upgrading your old tokens to the new style. Learn how in our <Hds::Link::Inline @href="{{env 'CONSUL_DOCS_URL'}}/guides/acl-migrate-tokens.html">documentation</Hds::Link::Inline>.</A.Description>
```

Comment:
![image](https://user-images.githubusercontent.com/91899352/221235393-a97c251f-e6be-436b-8992-ea7103acbce8.png)
Can we update to use the actions menu within the alerts component for all inline links? In this example instead of "Learn how in our documentation"  remove that text and add a link as shown in sreenshot "Learn more" with an external linking docs icon.

Reply:
By some chance I've missed that we planned to move to actions instead of using inline links. But actions looks much better, so happy to change that! 👍 
<img width="1203" alt="image" src="https://user-images.githubusercontent.com/10027860/221279910-bf913a17-b7a6-40e4-abb2-113f52171e0f.png">